library/src/main/java/com/xfinity/blueprint/presenter/DefaultComponentPresenter.kt

```diff
@@ -17,4 +17,19 @@ import com.xfinity.blueprint.view.ComponentView
 open class DefaultComponentPresenter : ComponentPresenter {
     override fun present(componentView: ComponentView<*>, componentModel: ComponentModel) {}
     override fun onComponentClicked(componentView: ComponentView<*>, position: Int) {}
+}
+
+@Suppress("UNCHECKED_CAST")
+abstract class DefaultComponentReflectionPresenter<in CV : ComponentView<*>, in CM : ComponentModel> : ComponentPresenter {
```

Comment:
If we're going to do this, maybe we should just use the generic at the ComponentPresenter level.